src/WebJobs.Script/Environment/EnvironmentSettingNames.cs

```diff
@@ -64,6 +64,7 @@ public static class EnvironmentSettingNames
         public const string FunctionsRequestBodySizeLimit = "FUNCTIONS_REQUEST_BODY_SIZE_LIMIT";
         public const string FunctionsHostIdCheckLevel = "FUNCTIONS_HOSTID_CHECK_LEVEL";
         public const string FunctionsPlatformConfigFilePath = "FUNCTIONS_PLATFORM_CONFIG_FILE_PATH";
+        public const string EnableLogsInHostV3 = "EnableLogsInHostV3";
```

Comment:
Nit -- can you put this with the other FeatureFlag constants in ScriptConstants.cs? Like next to `FeatureFlagEnableProxies`, for example?

Reply:
done